assets/icons/reload.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I trust that this will be minified somewhere along the build right?

Reply:
This SVG file is batch converted to an icon font by our framework; we are not using the SVGs embedded in the code.

Reply:
Great!